source/includes/entityref/_jobshiftassignment.md

```diff
@@ -0,0 +1,136 @@
+# JobShiftAssignment
+
+JobShiftAssignment ties the candidate to a JobShift and also allows for storing time capture details (such as actualStartTime and actualEndTime).
```

Comment:
Change to:
Ties the Candidate to a JobShift and also allows for storing time capture details such as actualStartTime and actualEndTime.